templates/chrony.conf.j2

```diff
@@ -45,3 +45,8 @@ commandkey 1
 generatecommandkey
 
 {% endif %}
+{% if timesync_max_distance != 0 %}
+# Limits which updates are accepted by this client.
```

Comment:
For consistency with other comments, can you please change it to the imperative style, e.g. "Limit maximum root distance"?

Reply:
done!